Makefile

```diff
@@ -7,10 +7,21 @@ release:
 	@git push --tags
 .PHONY: release
 
-build:
-	@gox -osarch="darwin/386 darwin/amd64 linux/386 linux/amd64 openbsd/386 openbsd/amd64 window/386 windows/amd64" -output="dnote-{{.OS}}-{{.Arch}}" ./...
+build: get-deps install-gox
+	@$(GOPATH)/bin/gox -osarch="darwin/386 darwin/amd64 linux/386 linux/amd64 openbsd/386 openbsd/amd64 window/386 windows/amd64" -output="dnote-{{.OS}}-{{.Arch}}" ./...
 .PHONY: build
 
+get-deps:
```

Comment:
I think specifically locking our deps to a version in the repository is a great idea. :+1: